pkg/fleets/controller_test.go

```diff
@@ -922,3 +1241,95 @@ func defaultGSSpec() *agonesv1.GameServerTemplateSpec {
 		},
 	}
 }
+
+func getAdmissionReview(raw []byte) admv1beta1.AdmissionReview {
+	gvk := metav1.GroupVersionKind(agonesv1.SchemeGroupVersion.WithKind("Fleet"))
+
+	return admv1beta1.AdmissionReview{
+		Request: &admv1beta1.AdmissionRequest{
+			Kind:      gvk,
+			Operation: admv1beta1.Create,
+			Object: runtime.RawExtension{
+				Raw: raw,
+			},
+		},
+		Response: &admv1beta1.AdmissionResponse{Allowed: true},
+	}
+}
+
+// MOCKS SECTION
+
+type fakeGSListerWithErr struct {
```

Comment:
Here and below if we have GameServerSet we are using GSS, GS is for GameServer
```suggestion
type fakeGSSListerWithErr struct {
```

Reply:
agree, thanks